Source/WebCore/css/CSSRayValue.h

```diff
@@ -36,32 +36,38 @@ namespace WebCore {
 // https://drafts.fxtf.org/motion-1/#funcdef-offset-path-ray.
 class CSSRayValue final : public CSSValue {
 public:
-    static Ref<CSSRayValue> create(Ref<CSSPrimitiveValue>&& angle, CSSValueID size, bool isContaining)
+    static Ref<CSSRayValue> create(Ref<CSSPrimitiveValue>&& angle, CSSValueID size, bool isContaining, RefPtr<CSSValue>&& positionX, RefPtr<CSSValue>&& positionY)
```

Comment:
I agree with @rreno's feedback from the other PR that we should just take and store the position as a CSSValuePair on CSSRayValue instead of 2 separate members.

Reply:
Fixed.